c/ztest.cc

```diff
@@ -27,6 +46,7 @@ static PKArgs arg_test_shmutex(3, 0, 0, false, false,
   "test_shmutex");
 
 static void test_shmutex(const PKArgs& args) {
+  check_posonly_args(arg_test_shmutex, args);
```

Comment:
`args` and `arg_test_shmutex` are actually exactly one and the same object here

Reply:
Oh, didn't notice that :)